### DISCUSSION TITLE
Multiline comments not displaying for which lines they are anymore

### DISCUSSION THREAD

**Discussion by BlindChickens** [community/community#28162] [Pull Requests]
This image displays what is missing, the small little comment in the red rectangle. We need that back please.

![image](https://user-images.githubusercontent.com/14092695/182583819-999baa7b-19ec-4304-8fa8-ee0f99f94d04.png)

I noticed the conversation tab will display the lines impacted above it when it is multiline. The problem is on the conversation tab a single line comment will display lines above it regardless, usually 4 lines, sometimes I see 5.
This causes the single line comment and multiline comment on the same line looknig exactly the same on the conversation tab.

I will attach an example of where I commented on the same line in the code. Once with a single line, and again with 4 lines selected. You will see that they look exactly alike.
![image](https://user-images.githubusercontent.com/14092695/182583654-1acdfb71-5a77-4258-a6b1-820a25f742a5.png)
![image](https://user-images.githubusercontent.com/14092695/182583671-40695452-916a-446d-b77a-24dcc621ff53.png)

If the comments did not specify which image refers to single/multi line, you would not be able to tell me.

If this is not frustration enough, the files tab is a completely different beast all together, because there you see lots of lines above comments all the time, this is where you can get more context of the code being reviewed because you can look at the code around it.

On the files tab it is even more important to have that small little comment showing what lines the comment is referring to.

That small comment was there in the past, and it got removed. Not exactly sure when. It is important to have that comment on the conversation tab as well as the files tab.

Please let me know if I misunderstood something.

**Comment by landonvg** [comment]
I'm seeing the same issue.  What does it take to get someone at Github to acknowledge this issue?

**Reply by BlindChickens** [reply]
Riots.

**Comment by kalkv** [comment]
Still the case.

Edit: confirmed on particular commits reviews or `See last changes` reviews, therefore it is probably a bug regarding diffs changes reviews

**Comment by alihoushngi** [comment]
I'm seeing the same issue , what is your idea
